src/article/editor/DownloadSupplementaryFileTool.js

```diff
@@ -39,6 +38,10 @@ export default class DownloadSupplementaryFileTool extends Tool {
       this.refs.link.el.attr({
         'href': url
       })
+      // Note: in the browser version we want to open remote files in a new tab
```

Comment:
Would be better to have this at the same location in code, as the `@download` thing.

Reply:
I have improved the implementation to be more clear.